docs/howto-mac.md

```diff
@@ -0,0 +1,61 @@
+# RLLIB SUMO Docker environment for MacOS
+
+## Step 1: Build your docker image
+
+Basic command to use:
+
+`bash docker-cmd-mac.sh --build`
+
+- The default tag name of this image is `tf-gpu-sumo-{today_date}`. You can also add option such as `--image-name={new_name}` to specify your own name;
+- The default path of the dockerfile to be used is `docker-image-mac`. You can adapt to your setting by using option `--image-folder`;
+- The default image building process does not use cache. You can enable it by adding the option `--cache`.
+
+## Step 2: Run a container of the built image
+
+`sudo bash docker-cmd-mac.sh --image-name=tf-gpu-sumo-2020-08-03 --run --no-gpu`
```

Comment:
Running docker using sudo it's not a good practice. 
Given that without it does not work, and it's a problem of user privilege between host and container, it needs to be highlighted as a security issue, at least in the HOWTO.

Reply:
Updated in the commit ad4dbcad7f44a2c7da93efb0981f04b0fcd7c54d